packages/frameworks-astro/src/api/[...auth].ts

```diff
@@ -0,0 +1,3 @@
+import { AstroAuth } from "../server.js"
+
+export const { GET, POST } = AstroAuth()
```

Comment:
this is not well supported when building for CF pages since the scope is defined per function for environment variables. https://github.com/nowaythatworked/auth-astro/pull/97/files is the fix that would allow the entire integration to work well with the API routes.